web/js/map-OpenLayers.js

```diff
@@ -567,8 +567,8 @@ $.extend(fixmystreet.utils, {
                     f.geometry = new_geometry;
                     this.removeAllFeatures();
                     this.addFeatures([f]);
-                    var qs = fixmystreet.utils.parse_query_string();
-                    if (!qs.bbox) {
+                    var qs = OpenLayers.Util.getParameters(fixmystreet.original.href);
+                    if (!qs.bbox && !qs.lat && !qs.lon) {
```

Comment:
I understand the lat/lon addition, but why check the original href and not what it is now?

Reply:
At this point I have totally forgotten my reasoning 😞 I think it was to deal with odd behaviour on `/reports`, but can't replicate it so will go back to `fixmystreet.utils.parse_query_string`.

Reply:
The Cypress failure now is because the map is no longer zooming out on a `/reports` page once the shapefile loads, because the URL has already updated by then to include location parameters - presumably that was the reason for this! It feels like there should be a better way of doing this, but I'm not sure what.